xml/Microsoft.JScript.Vsa/VsaEngine.xml

```diff
@@ -146,8 +146,8 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>This class, member, or property is internal to the script engine and should not be called from your code.</summary>
-        <returns>To be added.</returns>
+        <summary>Triggers compilation of all the state currently in the engine. All references, code items, host items, and resources are run through the compiler.</summary>
+        <returns><see langword="true" /> if the compilation was successful, otherwise, <see langword="false" />.</returns>
```

Comment:
```suggestion
        <returns>
        <see langword="true" /> if the compilation was successful; otherwise, <see langword="false" />.</returns>
```